content/en/docs/instrumentation/js/exporters.md

```diff
@@ -29,6 +29,7 @@ update `instrumentation.ts|js` from the
 following:
 
 <!-- prettier-ignore-start -->
+
```

Comment:
@svrnm - does textlint actually reformat too? I.e., did it add these newlines or were they added manually? I'd rather stick with the formatting that Prettier is ok with, but if this can't be avoided, so be it.

Reply:
See here: https://github.com/open-telemetry/opentelemetry.io/pull/2730#discussion_r1205368715

Reply:
Got it.